src/applications/vaos/tests/mocks/helpers.js

```diff
@@ -479,3 +479,42 @@ export function setupExpressCareMocks({
     numberOfRequests: isUnderRequestLimit ? 0 : 1,
   });
 }
+
+export function mockCommunityCareEligibility({
```

Comment:
I saw that you added some similar mocks, @narin. I had created this function while working on this PR a couple weeks ago, which mocks both requests in one place. Interested in if you think this is worth keeping or if I should switch to the functions you added.

Reply:
Let's switch to this since it can handle both requests in 1 shot